### ISSUE TITLE
Consider HJSON as the Standard SDF Input Format

### ISSUE THREAD

**Issue by gerickson** [one-data-model/language#2]
The lack of the ability to include comments and whitespace into JSON is a serious impediment to readability. While there is a "description" key, the inability to introduce whitespace and other as-is formatting into the values for "description" seems, at best, limiting.

The community seems to be generating some energy around [HJSON](https://hjson.org/). I would like to see ODM SDF adopt HJSON.

**Comment by akeranen** [comment]
I agree we need a way to include comments. HJSON looks pretty good but seems that we would always need to run the files through a HJSON pre-processor before using in plain JSON context and we would not have round-trip'ability back to HJSON. Not sure how big impediment would that be. Just for comments we could also use the JSON schema "$comment" keyword as discussed in #16.

**Comment by gerickson** [comment]
@akeranen, thank you for your comments of support on this front. The challenge I see with the "$comment" keyword or the "description" keyword and the way it is used is that there is no inclusion of white space, in particular newlines, which greatly aids and promotes human-readability.

**Comment by WAvdBeek** [comment]
one can use "\n" in the string, so that if we do an viewer it can space things out.
nicer would be if we can use markdown in those text descriptions.

note that some editors can do line breaks.

**Comment by mjkoster** [comment]
We could use txt2json:
https://github.com/mjkoster/txt2json/tree/master

**Comment by cabo** [comment]
A popular form of JSON with features for human use is YAML.  That includes JSON as a proper subset, so any JSON already is YAML.  It also has been around and stable for a decade, with many implementations.  It is not formally standardized, but neither are the other attempts at defining a humanly usable JSON derivative.

**Comment by mjkoster** [comment]
Although JSON is the "source of truth" for ODM, it would be good to provide a human readable format that is supported in our tools, so developers can submit definitions in the human-readable format, and maybe we can use it for reviews.
